recipes/binutils/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from conan import ConanFile
+from conan.tools.gnu import Autotools, AutotoolsDeps, AutotoolsToolchain
+from conans import tools
+from glob import glob
+
+required_conan_version = ">=1.44.0"
```

Comment:
we don't run 1.44 in CI yet. that's why `conan export` fails

@jgsogo I guess we need to improve an error message for that particular case

Reply:
Thanks @SSE4!